pages/images.jsx

```diff
@@ -15,26 +16,55 @@ import SeoConfig from '../src/models/SeoConfig';
 import { ImageGrid } from '../src/components/ImageGrid';
 import { ImagesStyles } from '../src/styles';
 import { getImages } from '../src/services/image';
-import { isAtEnd } from '../src/utils/helpers';
+import { SortController, isAtEnd, getQueryString } from '../src/utils/helpers';
+import { SortMappings } from '../src/constants';
 import { useInfiniteScroll } from '../src/hooks';
+import { SortSelect } from '../src/components/SortSelect';
 
 const useStyles = makeStyles(ImagesStyles);
 const pageTitle = 'Images';
 const pageSubtitle = 'Unfiltered list of all my favorite images.';
 
+/**
+ * Fetch a list of images.
+ *
+ * @param {string} sortQuery sort field and direction of request
+ * @param {number} limitQuery limit number of pagination request
+ * @param {number} pageQuery offset number of pagination request
+ */
+const fetchImages = async (sortQuery, limitQuery, pageQuery) => {
+  const queryDict = { sort: sortQuery, page: pageQuery, limit: limitQuery };
+  const route = `/api/images?${getQueryString(queryDict)}`;
+  const res = await fetch(route);
+  return res;
+};
+
 const Images = (props) => {
   const classes = useStyles();
   const imageGridRef = createRef();
   const {
-    appTitle, appEnv, rootUrl, hasError, images,
+    appTitle,
+    appEnv,
+    rootUrl,
+    hasError,
+    sortOptions,
+    defaultSort,
+    images,
   } = props;
   const {
     totalItems,
     limit,
     page,
     rows,
   } = images;
+  let sort = defaultSort.query;
```

Comment:
Should this be renamed so it is more clear? There are a lot of sort variables right now.